tests/testthat/test-table_egt05_qtcat.R

```diff
@@ -10,7 +10,8 @@ adeg_labels <- var_labels(adeg)
 adeg_f <- adeg %>%
   filter(
     PARAMCD == "QT",
-    ANL01FL == "Y"
+    ANL01FL == "Y",
+    EGTPT == 1
```

Comment:
```suggestion
   DTYPE == "AVERAGE"
```

Let's use the average value

Reply:
@shajoezhu if we filter by `DTYPE == "AVERAGE"` all rows have `CHG` values of `NA`, which consequently removes all of the "Change from Baseline" sections from the table

Reply:
hi @edelarua , lets keep your changes, per discussion with Edoardo.